src/palakSJun20/StringRotationCheck.java

```diff
@@ -0,0 +1,34 @@
+/*Coding test 10 :
+Print minimum rotation required to compare both strings equality.
+Example 2 :
+String input1 = "adbcdef"; -> push right
+fadbcde
+efadbcd
+defadbc
+*/
+package palakSJun20;
+
+public class StringRotationCheck {
+
+	public void findMinimumRotattion(String input1, String input2) {
+		int count = 0;
+		if (input1.length() != input2.length())
+			System.out.println("String lengths are not same");
+		else {
+			while (!input1.equals(input2)) {
```

Comment:
Just a suggestion. what if String can't be made equals even after all possible rotation ?? this will run infinitely ...

you can handled it by adding one more condition.
while(!input1.equals(input2) && count< input1.length)